associated-token-account/program/tests/spl_token_create.rs

```diff
@@ -0,0 +1,146 @@
+// Mark this test as BPF-only due to current `ProgramTest` limitations when CPIing into the system program
+#![cfg(feature = "test-bpf")]
+
+use {
+    solana_program::pubkey::Pubkey,
+    solana_program_test::*,
+    solana_sdk::{program_pack::Pack, signature::Signer, transaction::Transaction},
+    spl_associated_token_account::{
+        get_associated_token_address, id, instruction::create_associated_token_account,
+        processor::process_instruction,
+    },
+    spl_token::state::Account,
+};
+
+#[allow(deprecated)]
+use spl_associated_token_account::create_associated_token_account as deprecated_create_associated_token_account;
+
+fn program_test(token_mint_address: Pubkey, use_latest_spl_token: bool) -> ProgramTest {
+    let mut pc = ProgramTest::new(
+        "spl_associated_token_account",
+        id(),
+        processor!(process_instruction),
+    );
+
+    if use_latest_spl_token {
+        // TODO: Remove after Token >3.2.0 is available by default in program-test
```

Comment:
Update? Will it be 3.4?

Reply:
Ah right, the change will be in 3.4, so `> 3.3.0`